core/modules/translation/translation.install

```diff
@@ -62,6 +62,20 @@ function translation_update_1001() {
   }
 }
 
+/**
+ * Add "Show content translation links" setting to each content type.
+ */
+function translation_update_1002() {
+  $types = config_get_names_with_prefix('node.type.');
+  foreach ($types as $config_name) {
+    $config = config($config_name);
+    if ($config->get('show_translation_links') === NULL) {
+      $config->set('show_translation_links', TRUE);
```

Comment:
This doesn't look right to me. This looks like it's going to save the setting into the root of the config file, it should be under `settings.show_translation_links`.

Because this option is being provided by Translation module, it would also be great if we name-spaced this setting so that the word `translation_` came first. That would better line up with comment and menu module. So would suggest `translation_show_links` instead of `show_translation_links`.

Reply:
Thanks - I've made these changes.